integration-tests/helpers/admin-variants-seeder.js

```diff
@@ -137,49 +139,6 @@ module.exports = async (dataSource, data = {}) => {
     upc: "test-upc",
     barcode: "test-barcode",
     product_id: "test-product",
-    prices: [
```

Comment:
suggestion: depending on the result of the test about custom attr in pivot table, we would be able to revert this part maybe? keep me in touch